recipe/build.sh

```diff
@@ -1,11 +1,84 @@
-export CARGO_FEATURE_EXTERNAL_CPP=1
-export PSP_ROOT_DIR=$SRC_DIR/perspective-cpp
-cd perspective_python-${PKG_VERSION}
-cp -r ../perspective_python-${PKG_VERSION}.data .
-cd rust/perspective-client
-export CARGO_FEATURE_EXTERNAL_PROTO=1
-cargo build
-cd ../../
-unset CARGO_FEATURE_EXTERNAL_PROTO
-${PYTHON} -m pip install . -vv
-${PYTHON} ${RECIPE_DIR}/copy.py
+#!/bin/bash
+set -e
+
+# XXX(tom): don't understand why it's needed to set PKG_CONFIG_PATH.
+# this directory is already present in pkg-config's config path as reported by
+# `pkg-config --variable pc_path pkg-config` in a --debug build environment.
+# and yet `pkg-config <some-dependency>` fails
+export PKG_CONFIG_PATH=$BUILD_PREFIX/lib/pkgconfig/:$PKG_CONFIG_PATH
```

Comment:
still don't understand why this was needed but haven't looked at it in a minute, will self-review